css/zocial.css

```diff
@@ -134,7 +134,6 @@ a.zocial {
 .zocial.gmail,
 .zocial.instapaper,
 .zocial.itunes,
-.zocial.joinme,
```

Comment:
I followed the exact steps described

1 Install Font Custom
2 Add new font in the src/ folder.
3 Set color settings in the templates/zocial.css file.
4 Run fontcustom compile
5 Update the sample.html file with both the button and icon.
6 Test rendering. If broken go to step 2.
7 Send pull-request !

I only added the twitch line to the templates/zocial.css file, added the .svg to the src folder and ran "fontcustom compile"... So the compilation must have done all that.